module/enrichers.mjs

```diff
@@ -956,7 +956,7 @@ export function createRollLabel(config) {
         label = ability;
       }
       if ( config.passive ) {
-        label = game.i18n.format(`EDITOR.DND5E.Inline.DCPassive${longSuffix}`, { dc: config.dc, check: label });
+        label = game.i18n.format(`EDITOR.DND5E.Inline.${config.dc ? "DC" : ""}Passive${longSuffix}`, { dc: config.dc, check: label });
```

Comment:
You should use `showDC` here, since it takes the `hideDC` option into account:

```suggestion
        label = game.i18n.format(
          `EDITOR.DND5E.Inline.${showDC ? "DC" : ""}Passive${longSuffix}`, { dc: config.dc, check: label }
        );
```